llvm_ir_dataset_utils/compile_time_analysis_tools/python_processing/fitting_and_plotting.py

```diff
@@ -0,0 +1,72 @@
+"""plot_instruction_counts_v_textseg is used to make compiler instruction counts & text segment size scatter plots, and plot_instruction_counts_histograms is used to make compiler instruction counts histograms.
+
+plot_instruction_counts_v_textseg
```

Comment:
Function information should be included in function doc strings rather than in the module doc string.